tensorflow_lite_support/cc/task/processor/proto/nl_classification_options.proto

```diff
@@ -17,9 +17,6 @@ syntax = "proto2";
 
 package tflite.task.processor;
 
-option java_multiple_files = true;
-option java_package = "org.tensorflow.lite.task.processor.proto";
-
 // Options for setting up an NLClassifier.
 // Next Id: 7
 message NLClassificationOptions {
```

Comment:
This file is no longer necessary because you don't need to pass these config from Python layer to pybinds.

Reply:
@khanhlvg Removed it.